libbeat/autodiscover/providers/kubernetes/kubernetes.go

```diff
@@ -30,7 +30,8 @@ import (
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/libbeat/common/bus"
 	"github.com/elastic/beats/v7/libbeat/common/kubernetes"
-	"github.com/elastic/beats/v7/libbeat/keystore"
+	k8sks "github.com/elastic/beats/v7/libbeat/k8skeystore"
+	ks "github.com/elastic/beats/v7/libbeat/keystore"
```

Comment:
nit: let's not alias package names if possible. `keystore` is more readable throughout the file then `ks`.

Reply:
👍